tools/experiments/experiments.js

```diff
@@ -148,11 +148,12 @@ const EXPERIMENTS = [
     cleanupIssue: 'https://github.com/ampproject/amphtml/pull/6351',
   },
   {
-    id: 'ios-embed-wrapper',
+    id: 'ios-embed-sd',
```

Comment:
I think this has to be changed in a clean (no code changes) PR.

Reply:
No. Only prod-/canary-config.json restrictions require separate PR b/c they are propagated to binaries independently (e.g. via promote). This file is mostly irrelevant to any release procedure.